pom.xml

```diff
@@ -205,4 +212,20 @@
         </profile>
     </profiles>
 
+    <!-- You need ~/.m2/settings.xml for aws-maven to work -->
+    <distributionManagement>
+        <repository>
+            <id>fivetran-maven-release</id>
+            <url>s3://fivetran-maven/public/release</url>
+        </repository>
+    </distributionManagement>
+
+    <repositories>
+        <repository>
+            <id>aws-public-release</id>
+            <name>AWS Public Release Repository</name>
+            <url>https://s3.amazonaws.com/fivetran-maven/public/release</url>
+        </repository>
+    </repositories>
```

Comment:
same here, we won't publish to maven

Reply:
hm, i wonder where these came from, because they are fivetran-specific
###